charter.md

```diff
@@ -0,0 +1,105 @@
+# SIG Release Charter
+
+This charter adheres to the conventions described in the [Kubernetes Charter README] and uses
+the Roles and Organization Management outlined in [sig-governance].
+
+## Scope
+
+- Production of high quality Kubernetes releases on a reliable schedule
+- Ensure there is a consistent group of community members in place to support the release process
```

Comment:
how do we add `across time` concept here? one of the problems we have right now is that all sig meetings are mostly task based for the specific release and we hardly spend time on things that need to be done across releases.

Reply:
IMO, current RT topics should be contained to RT / Burndown meetings, except where it is required to clarify existing policy or help define new policy.